scss/components/_case-peoples.scss

```diff
@@ -17,14 +17,19 @@
 
   .select2-choice {
     height: 30px;
-    padding: 2px 0 2px 10px;
+    padding: 5px 0 5px 10px;
 
     .select2-arrow {
       right: 0;
       top: 0;
       width: 24px;
     }
 
+    b {
+      position: relative;
+      top: 2px;
+    }
```

Comment:
same as above: why is this style necessary only here, when select2 is used everywhere?

Reply:
Same as above. Updated the styles.